modules/applyPresentersToData.js

```diff
@@ -20,6 +29,7 @@ function presentDataSingle (collectionConfig, record, user, callback) {
 
     record = JSON.parse(JSON.stringify(record));
```

Comment:
This would not be needed if transformArraysProperties was not mutatative.

Reply:
Awesome